core/utils/file_utils.py

```diff
@@ -88,6 +89,20 @@ def get_size(folder):
                 total_size += os.path.getsize(file_path)
         return total_size
 
+    @staticmethod
```

Comment:
I believe we don't need `get_current_folder()` and `navigate_to()` methods, since `run` can take `cwd` as param and this eliminates the need methods above.